instructor/cli/usage.py

```diff
@@ -68,16 +83,6 @@ def get_model_cost(
     """Get the cost details for a given model."""
     if model in MODEL_COSTS:
         return MODEL_COSTS[model]
-
-    # Handle prefix-based matching
-    if model.startswith("gpt-3.5-turbo-16k"):
-        return MODEL_COSTS["gpt-3.5-turbo-16k"]
-    elif model.startswith("gpt-3.5-turbo"):
-        return MODEL_COSTS["gpt-3.5-turbo"]
-    elif model.startswith("gpt-4-32k"):
-        return MODEL_COSTS["gpt-4-32k"]
```

Comment:
the starts with for some of them without the prefix is needed since 

if they make `gpt-3.5-turbo-0327` it'll break